spotbugs-tests/src/test/java/edu/umd/cs/findbugs/TextUICommandLineTest.java

```diff
@@ -11,24 +11,34 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 class TextUICommandLineTest {
     @Test
-    void handleOutputFileReturnsRemainingPart() throws IOException {
+    void duplicateFilepathIsIgnored() throws IOException {
         Path file = Files.createTempFile("spotbugs", ".xml");
         TextUICommandLine commandLine = new TextUICommandLine();
-        SortingBugReporter reporter = new SortingBugReporter();
-        String result = commandLine.handleOutputFilePath(reporter, "withMessages=" + file.toFile().getAbsolutePath());
 
-        assertThat(result, is("withMessages"));
+        TextUIBugReporter call1 = commandLine.initializeReporter("withMessages=" + file.toFile().getAbsolutePath(),
+                () -> new SortingBugReporter(), (r, rOpts) -> {
```

Comment:
Can we not abbreviate here and use something closer to what the method says it is?  That helps in reading it in the future without having to attempt to figure it out.

Reply:
considering that these are NOOPs I didn't put much work into these names, but happy to update